src/patternfly/components/ContextSelector/context-selector.scss

```diff
@@ -37,12 +37,12 @@
   --pf-c-context-selector__menu--BoxShadow: var(--pf-global--BoxShadow--md);
 
   // Menu input
-  --pf-c-context-selector__menu-input--PaddingTop: var(--pf-global--spacer--sm);
-  --pf-c-context-selector__menu-input--PaddingRight: var(--pf-global--spacer--md);
-  --pf-c-context-selector__menu-input--PaddingBottom: var(--pf-global--spacer--md);
-  --pf-c-context-selector__menu-input--PaddingLeft: var(--pf-global--spacer--md);
-  --pf-c-context-selector__menu-input--BottomBorderColor: var(--pf-global--BorderColor--100);
-  --pf-c-context-selector__menu-input--BottomBorderWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-context-selector__menu-search--PaddingTop: var(--pf-global--spacer--sm);
+  --pf-c-context-selector__menu-search--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-context-selector__menu-search--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-context-selector__menu-search--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-context-selector__menu-search--BottomBorderColor: var(--pf-global--BorderColor--100);
```

Comment:
Nit - in this PR can we update `BottomBorder` vars to be `BorderBottom`?

Reply:
updated @mcoker